src/lib/getWikipediaInfo.ts

```diff
@@ -0,0 +1,109 @@
+import { Result } from './scrape';
+import * as got from 'got';
+import * as createFuzzySet from 'fuzzyset.js';
+import { first } from 'lodash';
+
+const WIKIPEDIA_API_ENDPOINT = 'https://en.wikipedia.org/w/api.php';
+
+export type WikipediaData = {
+  url: string;
+  title: string;
+  thumbnail: {
+    source: string;
+    width: number;
+    height: number;
+  };
+};
+
+export async function getWikipediaInfo(
+  result: Result,
+  thumbnailHeight = 300,
+): Promise<Partial<WikipediaData>> {
+  const urlRequest = got(WIKIPEDIA_API_ENDPOINT, {
+    json: true,
+    query: {
+      action: 'query',
+      generator: 'search',
+      gsrsearch: result.name,
+      gsrprop: 'snippet',
+      prop: 'info',
+      inprop: 'url',
+      format: 'json',
+    },
+  });
+
+  const imageRequest = got(WIKIPEDIA_API_ENDPOINT, {
+    json: true,
+    query: {
+      action: 'query',
+      titles: result.name,
+      prop: 'pageimages',
+      pithumbsize: thumbnailHeight,
+      format: 'json',
+    },
+  });
+
+  const urlBody = (await urlRequest).body as {
```

Comment:
It would've made it easier for me to understand this if it were called `pageResponseBody`